### PR TITLE
Add run-android command when print CLI usage

### PR DESCRIPTION
**Pull request by patoroco**
Add a new line to the print cli usage with run-android command.

### CODE REVIEW

File: local-cli/cli.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ function printUsage() {
     '  install: installs npm react components',
     '  bundle: builds the javascript bundle for offline use',
     '  new-library: generates a native library bridge',
-    '  android: generates an Android project for your app'
+    '  android: generates an Android project for your app',
+    '  run-android: open app in the Android simulator'
```
Comment (mkonicek) [inline local-cli/cli.js]: This is not necessarily a simulator, it can be a device. Also on Android these are called emulators.

**Comment by mkonicek** [comment]
Sorry for the delay! Was a bit overwhelmed with pull requests.

**Comment by mkonicek** [comment]
Removing assignment so this PR doesn't show up in my internal diff queue. I'll still get notifications for it.

**Comment by mkonicek** [comment]
Sorry for getting back to you late on this! Looks like this has been added in the meantime:
https://github.com/facebook/react-native/blob/master/local-cli/cli.js#L46

**Comment by patoroco** [comment]
Oh, don't worry, it's better late than never :P

**Pull request closed without merging**